src/Parameters/JoinMeetingParameters.php

```diff
@@ -40,14 +40,17 @@ class JoinMeetingParameters extends UserDataParameters
 
     private ?string $webVoiceConf = null;
 
-    private ?string $creationTime = null;
+    private ?float $creationTime = null;
```

Comment:
It might break existing integrations storing this value in the database without float transformation but not sure.

Reply:
The usage of `creationTime` was very in consistent before, sometimes `string`, sometimes `int` and in the PhpDoc (src/Responses/CreateMeetingResponse.php) it was mentioned float. The XML comes as `string` and must anyway be transformed into `int` or `float`. The usage inside of bigbluebutton-api is now consistent as `float`. I decided for `float` due to the [x-day issue of the UNIX-time](https://www.leebutterman.com/2021/02/01/store-your-unix-epoch-times-as-float64.html).

Since it was inconsistent before, we will break anyhow something... but now it is "technically" I guess the correct approach.